to_do.js

```diff
@@ -0,0 +1,22 @@
+const list = document.querySelector('.uList');
```

Comment:
We are going to work with this, but you have to place your logic within the DOMContentLoaded event listener.